packages/event-handler/src/appsync-graphql/RouteHandlerRegistry.ts

```diff
@@ -40,7 +40,13 @@ class RouteHandlerRegistry {
   public register(
     options: RouteHandlerOptions<Record<string, unknown>, boolean, boolean>
   ): void {
-    const { fieldName, handler, typeName, throwOnError, aggregate } = options;
+    const {
+      fieldName,
+      handler,
+      typeName,
+      throwOnError = false,
```

Comment:
The behaviour downstream will be subtly different now, default parameters will only only occur if the value is `undefined` whereas the the nullish coalescing operator used later would have caught both`null` and `undefined`. Don't think it's a issue but just wanted mention it.

Reply:
I see and you are right - customers should not pass `null` when registering the handler and TypeScript would complain since the parameters are set to `boolean | undefined`, however it might happen if you're using JavaScript.

Do you think I should make a commit to handle this? I could replace `throwOnError = false` with:

```ts
const {
  // other props
  throwOnError,
  // other props
} = props;

let actualThrowOnError = isNullOrUndefined(throwOnError) ? false : true;
```

what do you think?

Reply:
This issue has actually burned me in pure JS projects before but I would be quite surprised if people were passing `null` in this options object in a JS project, it feels very unnatural to me. I think we're OK leaving as is.